code/modules/mob/living/basic/bots/_bots.dm

```diff
@@ -814,6 +814,11 @@ GLOBAL_LIST_INIT(command_strings, list(
 /mob/living/basic/bot/spawn_gibs(drop_bitflags = NONE)
 	new /obj/effect/gibspawner/robot(drop_location(), src)
 
+/mob/living/basic/bot/get_hit_area_message(input_area)
```

Comment:
what about `/mob/living/simple_animal/bot`

Reply:
that's a good point i forgot about those

Reply:
i shake my fist at the sky